.ci/install_crio.sh

```diff
@@ -146,10 +146,10 @@ make BUILDTAGS="$(IFS=" "; echo "${build_union[*]}")"
 sudo -E install -D -m0755 runc "/usr/local/bin/crio-runc"
 popd
 
-echo "Set manage_network_ns_lifecycle to true"
-network_ns_flag="manage_network_ns_lifecycle"
+echo "Set manage_ns_lifecycle to true"
```

Comment:
I think it may be useful to have backward compatibility here. Wdyt @chavafg ?

Reply:
@amshinde adding this